cloudinit/config/cc_ssh_import_id.py

```diff
@@ -43,6 +43,7 @@
             """
         )
     ],
+    "activate_by_schema_keys": [],
```

Comment:
The only reason we can't provide `ssh_import_id` here is because we have a nested check for ssh_import_id under users definitions via is_key_in_nested_dict. If we grew support for callables we'd be able to disregard if is_key_in_nested_dict("ssh_import_id") but I don't think this one case is worth growing that feature. If we have multiple cases that align, it may make sense to do something a bit more complex.